charts/envoy/values.yaml

```diff
@@ -32,13 +32,22 @@ podSecurityContext: {}
   # fsGroup: 2000
 
 # securityContext -- Setting security context at the pod applies those settings to all containers in the pod
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+securityContext:
```

Comment:
Sorry, I'm a bit confused. 
`securityContext` defined here seems not referred whereas `podSecurityContext` above is empty but it seems to be referred by the following.
 Is it OK?
https://github.com/scalar-labs/helm-charts/pull/49/files#diff-f9e366b1cdbdff67956aecad7953e831aed6c1bc6e0b798f3c5fa59845bb5983R31

Reply:
@feeblefakie 

Yes, the above configuration is fine.

The corresponding security context for the entire pod is declared as `PodSecurityContext`.
In contrast, the `SecurityContext` is the Security Context of an individual Container.
If there is any overlap, the container's security context will take precedence.

Also, the properties that can be set are different between the two, and the container's security context has more properties that can be set.
In this case, the security context of the container is used.

There is currently only one scalar-envoy container, so we believe that it can meet the requirements.

Reply:
Thank you!